app/src/main/java/com/aileyzhang/musicapp/fragments/AlbumListFragment.java

```diff
@@ -0,0 +1,35 @@
+package com.aileyzhang.musicapp.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ListView;
+
+import com.aileyzhang.musicapp.R;
+import com.aileyzhang.musicapp.SongData;
+
+/**
+ * Created by Ailey on 2017-12-19.
+ */
+
+public class AlbumListFragment extends android.support.v4.app.Fragment {
+    private SongData songData;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        // Initialize song data
+        songData = new SongData(getContext());
```

Comment:
I know this (and a lot of things right now) are temporary, but rethink if you want multiple instances of this. Also actually since this is the album list fragment, I think you wouldn't wanna initialize the song data stuff here specifically, but you can leave the one in the song list 👌

Reply:
So should I just remove it here

Reply:
Yes